internal/database/sqlite_migrate.go

```diff
@@ -1422,7 +1423,10 @@ ALTER TABLE filter_external_dg_tmp
 	`ALTER TABLE filter_external
 	DROP COLUMN webhook_retry_max_jitter_seconds;
 `,
-  `ALTER TABLE irc_network
+	`ALTER TABLE irc_network
 	ADD COLUMN bot_mode BOOLEAN DEFAULT FALSE;
 	`,
+	`ALTER TABLE action
+    ADD COLUMN action_priority TEXT;
```

Comment:
Probably not the best name for it? 🤔

Reply:
Maybe just priority is enough since the table is action already. Becomes like this otherwise `actions.actions_priority`.

Reply:
It might be me overlooking something, but I had it as `priority` initially, but then it errored on expecting int64 somewhere, probably due to `filters.priority`. I can try again later tonight though!